src/cloudant/__init__.py

```diff
@@ -24,30 +21,69 @@
 
 from .account import Cloudant, CouchDB
 
-
 @contextlib.contextmanager
 def cloudant(user, passwd, **kwargs):
     """
-    _cloudant_
-
-    Context helper to create a cloudant session and
+    Context helper to create a Cloudant session and
```

Comment:
Changed to "Context manager".  Done in 5b504f9e843733440e09b22a47adf271bb246bc9.